package.json

```diff
@@ -11,7 +11,8 @@
     "fast-glob": "^3.2.7",
     "husky": "^7.0.4",
     "lint-staged": "^12.3.4",
-    "lodash": "4.17.21"
+    "lodash": "4.17.21",
+    "prettier": "npm:wp-prettier@2.6.2"
```

Comment:
Why do we need to add `prettier` here?

Reply:
When i run `npm run lint-js` it show `Error: Cannot find module 'prettier'` error. Similar error for the JS lint https://github.com/WordPress/performance/runs/7733913318?check_suite_focus=true

Reply:
I downgraded NPM to 14 and am no longer experiencing this error. 

Reply:
@mukeshpanchal27 
> I downgraded NPM to 14 and am no longer experiencing this error. 

Shouldn't we remove the addition of `prettier` here then? FWIW I checked out this branch and removed `prettier` from here, however it gives me another error than yours: `ESLint couldn't find the plugin "eslint-plugin-prettier".`

@eugene-manuilov Just double-checking with you, does this look reasonable to you adding this dependency here? It just seemed a bit unrelated to the underlying problem we're solving here, that's the main reason I flagged it.

Reply:
> Just double-checking with you, does this look reasonable to you adding this dependency here? It just seemed a bit unrelated to the underlying problem we're solving here, that's the main reason I flagged it.

@felixarntz IMO we should revert `package.json` changes as they are irrelevant to this PR. I tried it on my end and i didn't have prettier problem on the `trunk` branch.

> I downgraded NPM to 14 and am no longer experiencing this error.

@mukeshpanchal27, we use nvm as our nodejs version manager. The version defined in the `.nvm` file is the version that we need to use when we work on this project. Please, install nvm if you don't have it yet, and use it to have the correct version of node.

Reply:
Thanks for the feedback.

@felixarntz @eugene-manuilov If i remove `prettier` from the package.json and if i run `npm run lint-js` then it show error `Error: Cannot find module 'prettier'`. Can you please check if you replicate it?

Reply:
@mukeshpanchal27 I can't reproduce it on my end. Probably something wrong is configured on your end. Use nvm to switch to the 14th version of node and then re-install npm dependencies with `npm ci` command. I think it should help. Let me know if it doesn't.

Reply:
Thanks @eugene-manuilov for the reply.

I tried what you suggested but it still show me the error. Please check the attached screenshot for more info.

<img width="1720" alt="Screenshot 2022-08-31 at 1 52 26 PM" src="https://user-images.githubusercontent.com/10103365/187632307-796606fd-62a6-44eb-8d5c-037c97009316.png">

My node version it 14. When i run `node -v` it show `v14.17.3`

I see some warning for manual instalation.

```
npm WARN acorn-import-assertions@1.8.0 requires a peer of acorn@^8 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-prettier@3.4.1 requires a peer of prettier@>=1.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@3.21.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
```

Reply:
It's working now. Thanks